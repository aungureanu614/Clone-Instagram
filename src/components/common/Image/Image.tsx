import * as React from 'react'

interface Props {
  image: string
  alt: string
  className?: string
}

const Image: React.FunctionComponent<Props> = ({ image, alt, className }) => (
  <img src={image} alt={alt} className={className} />
)

export default Image
