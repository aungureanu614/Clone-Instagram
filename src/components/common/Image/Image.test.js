import React from 'react'
import renderer from 'react-test-renderer'
import Image from './Image'
import image from '../../../images/ig-logo.png'

test('Image renders', () => {
  const component = renderer.create(
    <Image image={image} alt="Instagram Icon" className="Icon" />
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
