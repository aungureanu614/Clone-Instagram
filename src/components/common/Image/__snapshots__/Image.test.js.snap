// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image renders 1`] = `
<img
  alt="Instagram Icon"
  className="Icon"
  src="ig-logo.png"
/>
`;
